viewing_party/party.py

```diff
@@ -1,23 +1,156 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if (not title) or (not genre) or (not rating):

```

Comment:
nice job! syntactic-wise, it is standard practice to not use parentheses like this. It certainly might make it easier to read in some regards, but you don't see it like this. Likely another coder might think you have a background in JavaScript and forgot to code switch correctly.

```suggestion
    if not title or not genre or not rating:
```